basics/searching-packages.md

```diff
@@ -0,0 +1,18 @@
+# Searching packages
+
+The simplest way to find a package is to use the web frontend on the
+NixOS website: https://nixos.org/nixos/packages.html
+
+On the command line the `nix-env` command supports not installation
```

Comment:
i would rephrase it totally as pretty much every package manager command i know both supports installation and search